README.md

```diff
@@ -11,48 +11,204 @@
 composer require keboola/csvmap
 ```
 
-## Configuration
+## Usage
```

Comment:
1. `Better documentation.`

Dopnil som dokumentaciu, ... v tej povodnej chybali priklady, aby sa to dalo jednoducho pochopit.